javalib/src/main/scala/java/util/Formatter.scala

```diff
@@ -379,6 +382,9 @@ final class Formatter private (private[this] var dest: Appendable,
           case arg: Long =>
             padAndSendToDest(RootLocaleInfo, flags, width, prefix,
                 java.lang.Long.toOctalString(arg))
+          case arg: BigInteger =>
+            formatNumericString(RootLocaleInfo, flags, width,
+              arg.toString(8), prefix)
```

Comment:
We use 4 spaces of indentation for continuation lines:
```suggestion
            formatNumericString(RootLocaleInfo, flags, width,
                arg.toString(8), prefix)
```